docs/data-access/data-download.md

```diff
@@ -80,31 +140,187 @@ metadata.url=https://song.rdpc.cancercollaboratory.org
 
+- **--object-id** : object-id of the file to be downloaded
+- **--output-dir**: directory location you want the downloaded files to be written to
+
+For example: 
 ```shell
-> METADATA_URL=https://api.platform.icgc-argo.org/storage-api STORAGE_URL=https://api.platform.icgc-argo.org/storage-api score-client download --manifest manifest1.txt
+> score-client-/bin/score-client download --objectid ce86a332-407a-11eb-b378-0242ac130002 --output-dir ./output-directory-path
+```
+
+You can also specify multiple object id's separated by spaces:
+```shell
+> bin/score-client download --object-id ddcdd044-adda-5f09-8849-27d6038f8ccd 5cc35183-9291-5711-967d-30afcf20e71f --output-dir data
+```
+
+
+
+
+## BAM/CRAM Slicing 
+
+The view command is a minimal version of [samtools](http://www.htslib.org/doc/samtools.html) view. You can view a “genomic slice” of the remote BAM file, freeing the user from having to download the entire file locally, saving bytes and time.
+
+The following example will download reads overlapping the region 1 - 10,000 on chromosome 1:
+
+```shell
+> bin/score-client view --object-id ddcdd044-adda-5f09-8849-27d6038f8ccd --query 1:1-10000
+```
+
+The BAI is automatically discovered and streamed as part of the operation. For quickly accessing only the BAM header one can issue:
+
+```shell
+> bin/score-client view --header-only --object-id ddcdd044-adda-5f09-8849-27d6038f8ccd
+```
+
+It is also possible to pipe the output of the above to `samtools`, etc. for pipelining a workflow:
+
+```shell
+> bin/score-client view --object-id ddcdd044-adda-5f09-8849-27d6038f8ccd --query 1:1-100000 | samtools mpileup -
+```
+
+
+## FUSE Mounting 
+
+The mount command can be used to mount the remote S3 bucket as a read-only  FUSE file system. This is very useful to browse and explore the available files, as well as quickly see their size and date of modification using common commands such as ls, find, du and tree. It also works very well with standard analysis tools such as `samtools`.
+
+In order to use the mount feature, `FUSE` is required. On most Linux based systems, this will require installing `libfuse-dev`, `fuse` and other packages. Below is the command to install them on Ubuntu.
+
+``` shell
+> sudo apt-get install -y libfuse-dev fuse curl wget software-properties-common
+```
+
+
+Files are organized into a virtual directory structure. The following shows the default bundle layout:
+```
+/bundleId1/fileName1
+/bundleId1/fileName2
+...
+/bundleId1/fileNamei
+...
+/bundleIdn/fileName1
+/bundleIdn/fileName2
+...
+/bundleIdn/fileNamej
 ```
 
-//// need details of if the ENV means you have to define aceess token as an env variable???
+where `bundleId` and `fileName` are the original Bundle ID and file name of the file respectively. It possible to control the layout using the `--layout` option. Using `--layout object-id` will instead produce a flat list of files named by their associated object id.
+
```

Comment:
Do you think it is better to combine the above example of `virtual directory structure` with the following concrete examples?

Reply:
linda, that would be a good idea.  unfortunatley i cant seem to get this working yet, so i need to understand a bit more how to do that!